package.json

```
@@ -24,8 +24,8 @@
     "test:fix": "npm run test:prettier:fix && npm run test:eslint:fix",
     "test:eslint": "eslint 'src/**/*.{js,vue}' --cache",
     "test:eslint:fix": "npm run test:eslint -- --fix",
-    "test:stylelint": "stylelint 'src/**/*.scss' --config node_modules/do-bulma/.stylelintrc.json",
-    "test:i18n-packs": "node --import=extensionless/register src/nginxconfig/i18n/verify.js",
+    "test:stylelint": "stylelint 'src/**/*.{scss}' --config node_modules/do-bulma/.stylelintrc.json",
```

```suggestion
    "test:stylelint": "stylelint 'src/**/*.scss' --config node_modules/do-bulma/.stylelintrc.json",
```
Should fix the stylelint error